napps/kytos/of_topology/openapi.yml

```diff
@@ -4,8 +4,9 @@ info:
   version: 0.1.1
   description: "**Warning**: *This documentation is experimental and will change soon.*\n
 
-  Keeps track of links between hosts and switches. Returns a JSON
-  with network topology using the endpoint /legacy/topology."
+  Keeps track of links between hosts and switches. Returns a JSON with network
+  topology using the endpoint /api/legacy/of_topology/topology.
+  "
```

Comment:
Keep track ... and return a JSON ...

Note: use imperative like in docstrings.